integration/util.go

```diff
@@ -217,6 +220,15 @@ func NewJaegerGRPCClient(endpoint string) (*jaeger_grpc.Reporter, error) {
 	return jaeger_grpc.NewReporter(conn, nil, logger), err
 }
 
+func NewSearchGRPCClient(endpoint string) (tempopb.StreamingQuerierClient, error) {
+	clientConn, err := grpc.DialContext(context.Background(), endpoint, grpc.WithTransportCredentials(insecure.NewCredentials()))
```

Comment:
It might be nice to allow for configuring the transport for things like TLS.

Reply:
that will remain on the long list :). since this is just a simple integration test function it's fine for now.